components/asana/sources/new-completed-task/new-completed-task.mjs

```diff
@@ -0,0 +1,83 @@
+import asana from "../../asana.app.mjs";
+
+export default {
+  key: "asana-new-completed-task",
+  type: "source",
+  name: "New Completed Task",
```

Comment:
Sources that use webhooks should have the word "Instant" at the end of the name in parenthesis.
```suggestion
  name: "New Completed Task (Instant)",
```